### PR TITLE
added change

### PR DESCRIPTION
**Pull request by tobiasmed**
pleas review my code

### CODE REVIEW

**Review by TobiasGasteiger — COMMENTED** [review]
This no good

**Comment by tobiasmed** [comment]
bettwer now

File: scans/free-from-not-dynamic-alloc-memory.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"errors": [], "interfile_languages_used": [], "paths": {"scanned": ["Damn_Vulnerable_C_lib/imgReadlib.h", "tinycrypt-master/lib/include/tinycrypt/aes.h", "tinycrypt-master/lib/include/tinycrypt/cbc_mode.h", "tinycrypt-master/lib/include/tinycrypt/ccm_mode.h", "tinycrypt-master/lib/include/tinycrypt/cmac_mode.h", "tinycrypt-master/lib/include/tinycrypt/constants.h", "tinycrypt-master/lib/include/tinycrypt/ctr_mode.h", "tinycrypt-master/lib/include/tinycrypt/ctr_prng.h", "tinycrypt-master/lib/include/tinycrypt/ecc.h", "tinycrypt-master/lib/include/tinycrypt/ecc_dh.h", "tinycrypt-master/lib/include/tinycrypt/ecc_dsa.h", "tinycrypt-master/lib/include/tinycrypt/ecc_platform_specific.h", "tinycrypt-master/lib/include/tinycrypt/hmac.h", "tinycrypt-master/lib/include/tinycrypt/hmac_prng.h", "tinycrypt-master/lib/include/tinycrypt/sha256.h", "tinycrypt-master/lib/include/tinycrypt/utils.h"]}, "results": [], "skipped_rules": [], "version": "1.78.0"}
```
Comment (TobiasGasteiger) [inline scans/free-from-not-dynamic-alloc-memory.json:1]: Fix this

Comment (TobiasGasteiger) [reply scans/free-from-not-dynamic-alloc-memory.json:1]: Good

**Review by TobiasGasteiger — APPROVED** [review]
Done

**Pull request merged**